go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/hudl/fargo v1.3.0
 	github.com/influxdata/influxdb1-client v0.0.0-20191209144304-8bf82d3c094d
 	github.com/lightstep/lightstep-tracer-go v0.22.0
-	github.com/nats-io/nats-server/v2 v2.2.6
+	github.com/nats-io/nats-server/v2 v2.2.6 // indirect
```

Comment:
Still a dependency, I think?

Reply:
Could be, but not a direct one at least. This is why I decided to leave that argument and wait if Go 1.17 improves the situation. I can't say I fully understand the relevant proposal yet.

Reply:
Gotcha. Just for the record, no real difference in indirect vs. direct from the "does my module depend on this" perspective.

Reply:
Not yet anyway. That's what Go 1.17 is supposed to address, but if nats server is only used in tests in kit it should probably be okay then.